x/marker/types/marker.go

```diff
@@ -228,6 +233,20 @@ func ValidateGrantsForMarkerType(markerType MarkerType, grants ...AccessGrant) e
 	return ValidateGrants(grants...)
 }
 
+// ValidateRequiredAttributes checks that required attributes are of the correct format
+func ValidateRequiredAttributes(requiredAttributes []string) error {
```

Comment:
Should `MarkerAccount.Validate()` call this?

Reply:
I was actually meaning to remove this.  The attribute validation is a little more complex than I first thought.    The validation will happen when the attributes are being normalized in the msg_server.  
```
	normalizedReqAttrs, err := k.NormalizeRequiredAttributes(ctx, msg.RequiredAttributes)
	if err != nil {
		return nil, err
	}
```